packages/framework/src/Twig/NumberFormatterExtension.php

```diff
@@ -54,11 +54,11 @@ public function getFilters()
     }
 
     /**
-     * @param mixed $number
+     * @param string $number
      * @param string|null $locale
      * @return string
      */
-    public function formatNumber($number, $locale = null)
+    public function formatNumber(string $number, ?string $locale = null): string
```

Comment:
is it ok that you are now forced to use string only? I thought that these filters could be used with various types (float, int, string)?

Reply:
We decided to keep it as it is because it is better to let developer to parse number correctly to string and sent it here.